pyproject.toml

```diff
@@ -327,12 +327,12 @@ channel-priority = "strict"
 name = "pudl-dev"
 
 [tool.conda-lock.dependencies]
-google-cloud-sdk = ">=452"
+google-cloud-sdk = ">=464"
```

Comment:
464 is the first version of the google-cloud-sdk that works with Python 3.12 (thanks to my prodding...)